sktime/datasets/_fpp3_loaders.py

```diff
@@ -0,0 +1,271 @@
+"""Utilities for loading datasets."""
+
+__author__ = [
+    "ericjb",
+]
+
+__all__ = [
+    "load_fpp3",
+]
+
+import os
+# import zipfile
+# from urllib.error import HTTPError, URLError
+# from warnings import warn
+
+import shutil
+import tarfile
+import tempfile
+import warnings
+import pandas as pd
+import requests
+import rdata
+
```

Comment:
The above two lines are expecting the soft dependencies to be installed already, if I am not wrong. These imports should be moved into functions/methods protected by checking for soft dependencies first.